datasets.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pandas as pd
+import zipfile
+import json
+
+
+def _download_dataset(url, zip_file, check_existence, output_dir):
```

Comment:
do we really need check_existence? can we just use the existence of the zip file to decide whether to download _and_ unzip?

Reply:
I might be overthinking this but the reason why I did it this way is that if the zip file exists but for whatever reason the user accidentally renames the name of the file we care about or modifies the unzipped folder in some weird way, the demo will not work and it might confuse them. This happened to me several times